integreat_cms/cms/views/media/media_actions.py

```diff
@@ -354,6 +354,22 @@ def delete_file_ajax(
         MediaFile.objects.filter(region=region), id=request.POST.get("id")
     )
 
+    # Check if the media file is in use
+    if media_file.is_used:
+        return JsonResponse(
+            {
+                "messages": [
+                    {
+                        "type": "warning",
+                        "text": _(
+                            'File "{}" cannot be deleted because it is used as icon or content.'
```

Comment:
```suggestion
                            'File "{}" cannot be deleted because it is used as an icon or content.'
```
"als Icon" works perfectly fine in German, but in English you need an indefinite article here, I am 99% sure